hw/sensor/syscfg.yml

```diff
@@ -40,3 +40,7 @@ syscfg.defs:
     SENSOR_OIC_OBS_RATE:
         description: 'Set OIC server observation rate in milli seconds'
         value: 1000
+
+    SENSOR_MGR_EVQ:
+        description: 'Specify the eventq to be used by sensor mgr'
+        value: 0
```

Comment:
Should the default value of `SENSOR_MGR_EVQ` be blank, rather than `0`?  The code uses `#ifdef` to check this setting, and a value of 0 is considered defined.  You can specify an empty value like this:
```
    SENSOR_MGR_EVQ:
        description: 'Specify the eventq to be used by sensor mgr'
        value:
```